controllers/component_build_controller.go

```diff
@@ -626,6 +627,19 @@ func GeneratePipelineRun(component appstudiov1alpha1.Component, bundle string, o
 		proposedImage = image_repo + ":{{revision}}"
 	}
 
+	params := []tektonapi.Param{
+		{Name: "git-url", Value: tektonapi.ArrayOrString{Type: "string", StringVal: "{{repo_url}}"}},
+		{Name: "revision", Value: tektonapi.ArrayOrString{Type: "string", StringVal: "{{revision}}"}},
+		{Name: "output-image", Value: tektonapi.ArrayOrString{Type: "string", StringVal: proposedImage}},
+	}
+	if dockerFile, err := devfile.SearchForDockerfile([]byte(component.Status.Devfile)); err == nil && dockerFile != nil {
+		if dockerFile.Uri != "" {
+			params = append(params, tektonapi.Param{Name: "dockerfile", Value: tektonapi.ArrayOrString{Type: "string", StringVal: dockerFile.Uri}})
+		}
+		if dockerFile.BuildContext != "" {
+			params = append(params, tektonapi.Param{Name: "path-context", Value: tektonapi.ArrayOrString{Type: "string", StringVal: dockerFile.BuildContext}})
+		}
+	}
```

Comment:
This ignores any potential err returned from `SearchForDockerfile`. Should it be handled? In this case, from build-service perspective, what kind of err should be handled or even blocks the build process?

Reply:
Separated into PLNSRVCE-642